### PR TITLE
Add the 'Research' section on my personal README.md

### PR DESCRIPTION
**Pull request by Chiemezuo**
I also ticked some more boxes too.

### CODE REVIEW

**Comment by codewithmide** [comment]
Hi @Chiemezuo, well done for submitting your PR

However, I'll suggest that you read through the [Contribution guide](https://github.com/wagtail/outreachy/blob/main/contributor-guide.md) once again and follow the tasks accordingly. 

Also, make sure to follow the Markdown style guide [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings)

If you use VScode, I suggest you download this extension [Markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint). It will help guide and fix your markdown file.

**Comment by Chiemezuo** [comment]
I went through those. What did I miss, though?

**Comment by codewithmide** [comment]
> I went through those. What did I miss, though?

It seems your `## Research` section is above the `## Task` section.

**Comment by Chiemezuo** [comment]
Oh. They weren't specific about where in the file to put that. I put it above for it to be easier to see. Or should there be a specific order @thibaudcolas ?

File: 2023/Akujobi-Chiemezuo/README.md [hunk 1]
```diff
@@ -13,6 +13,29 @@ For Outreachy, this will be my first time participating, but I've read and follo
...
+- **Wagtail's [slack](https://github.com/wagtail/wagtail/wiki/Slack)**: If you want to go fast, you could try going alone, but if you really want to go far, you should definitely join the Slack. Sometimes, I see answers to questions I didn't know I had. On one occasion, helping someone with pull requests helped me strengthen my understanding of concepts. Some other times, when I want to help with a question asked in a public channel, I read the comments thus far and see an even better solution than I would have offered, which makes me learn something new.
+- **This [guide](https://opensource.guide/how-to-contribute/) on how to contribute**: I mean, it goes without saying 😄.
```
Comment (thibaudcolas) [inline 2023/Akujobi-Chiemezuo/README.md:29]: :)

**Review by thibaudcolas — APPROVED** [review]
Nice one @Chiemezuo

File: 2023/Akujobi-Chiemezuo/README.md (hunk 1 above)
Comment (Chiemezuo) [reply 2023/Akujobi-Chiemezuo/README.md:29]: Thank you. 😅

**Pull request merged**